modules/Bio/EnsEMBL/Compara/RunnableDB/Ortheus.pm

```diff
@@ -438,63 +540,53 @@ sub parse_results {
     pop(@$ids); ## ...except for the last leaf which is the end of the tree
     #print join(" :: ", @$ids), "\n\n";
 
-    while (<$fh>) {
-	next if (/^\s*$/);
-	chomp;
-	## FASTA headers correspond to the tree and the order of the leaves in the tree corresponds
-	## to the order of the files
-
-	if (/^>/) {
-	    print "PARSING $_\n" if ($self->debug);
-	    my ($name) = $_ =~ /^>(.+)/;
-	    if (defined($this_genomic_align) and  $seq) {
-		    print "add aligned_sequence " . $this_genomic_align->dnafrag_id . " " . $this_genomic_align->dnafrag_start . " " . $this_genomic_align->dnafrag_end . "\n" if $self->debug;
+    my $parser = Bio::EnsEMBL::IO::Parser::Fasta->open($alignment_file);
+    while ($parser->next()) {
+        my $name = $parser->getHeader;
+        my $seq  = $parser->getSequence;
 
+		    my $this_genomic_align = new Bio::EnsEMBL::Compara::GenomicAlign;
 		    $this_genomic_align->aligned_sequence($seq);
 		    $this_genomic_align_block->add_GenomicAlign($this_genomic_align);
```

Comment:
```suggestion
        my $this_genomic_align = new Bio::EnsEMBL::Compara::GenomicAlign;
        $this_genomic_align->aligned_sequence($seq);
        $this_genomic_align_block->add_GenomicAlign($this_genomic_align);
```
I know those two lines "are not yours" but I think it may be worth for the sake of readability to change them.

Reply:
Regarding the indentation of this file, it is best seen with tab=8 spaces, but they do indentation levels of 4, so (1) 4 spaces, (2) 1 tab, (3) 1 tab and 4 spaces, (4) 2 tabs, etc :roll_eyes:
When changing a line, I have kept the original indentation, and when adding a line, I have matched the indentation of the surrounding lines. If we're fine with resetting the indentation to 4 spaces and no tabs (I'm fine with it), that would edit 8 lines, and I would rather go back to my own commits and edit them so that they use the correct indentation right from the start

Reply:
I would vote for the change if you don't mind, since I consider it an improvement. But will wait for the vote from the other reviewer.

Reply:
I'm not fussy